meilisearch-http/src/index/updates.rs

```diff
@@ -43,6 +43,9 @@ pub struct Settings {
         skip_serializing_if = "Option::is_none"
     )]
     pub ranking_rules: Option<Option<Vec<String>>>,
+
+    // TODO we are missing the stopWords, synonyms and distinctAttribute for the GET settings
+    // request
```

Comment:
What should we do here?
There is some fields we don't implement currently:
https://docs.meilisearch.com/reference/api/settings.html#get-settings

Reply:
Is it related to this PR? We only want to update the body response of the updates